packager/packager.cc

```diff
@@ -591,7 +592,8 @@ struct PackageInfoFinder {
 
 // Sanity checks package files, mutates arguments to export / export_methods to point to item in namespace,
 // builds up the expression injected into packages that import the package, and codegens the <PackagedMethods>  module.
-unique_ptr<PackageInfo> getPackageInfo(core::MutableContext ctx, ast::ParsedFile &package) {
+unique_ptr<PackageInfo> getPackageInfo(core::MutableContext ctx, ast::ParsedFile &package,
+                                       vector<std::string> extraPackageFilesDirectoryPrefixes) {
```

Comment:
Instead of copying `extraPackageFilesDirectoryPrefixes` on a call to this function, can we pass it as `const &`?

Reply:
Aha, I'm still building my intuition transitioning from call-by-reference to call-by-value semantics :). Will make this a pass by reference.